### ISSUE TITLE
Are the depthmaps in the OmniWorld-Game subset in the metric space?

### ISSUE THREAD

**Issue by ShenhanQian** [yangzhou24/OmniWorld#3]
Thank you for releasing this amazing dataset! You mentioned in the paper that you have finetuned DUSt3R and CUT3R with OmniWorld-Game. I am wondering if you treat the depthmaps as metric or not. This influeces how you normalize the predicted point maps.

Looking forward your reply!

**Comment by ShenhanQian** [comment]
Thanks for your reply!

**Comment by yangzhou24** [comment]
Hi @ShenhanQian, 

Extremely distant points can indeed make the training unstable. My approach is to filter out the depth values of extremely far points when reading the depth map. Refer to depth minimal reader in hugging face, you can set additional `far_mask` after filter sky like this:

`far_mask = depthmap > np.percentile(depthmap[~far_mask], 95)  # 3. filter far area (optional)`

This can improve the results to some extent. The filtering threshold can be adjusted depending on the specific situation.

**Comment by yangzhou24** [comment]
Hi @HLinChen ,

1. About the Depth Linearization Formula
The first part of the expression, `depthmap / (far - depthmap * (far - near))`, is indeed used to linearize the non-linear depth buffer that Reshade provides. The `/ 0.004` is simply a numerical scaling factor for stability.

2. About the CUT3R Fine-tuning Scale Issue
GT point cloud can sometimes contain extremely distant points, especially in large-scale outdoor scenes. These outliers can influence the scale estimation. Refer to depth minimal reader in hugging face, you can set additional `far_mask` after filter sky like this:`far_mask = depthmap > np.percentile(depthmap[~far_mask], 95)  # 3. filter far area (optional)`
 The filtering threshold can be adjusted depending on the specific situation.

3. About Point Cloud Visualization
 I download scene_id = ffdb6111904d from hugging face and visualize the split_0 use [main/scripts/visualize_pcd.py](https://github.com/yangzhou24/OmniWorld/blob/main/scripts/visualize_pcd.py?rgh-link-date=2025-09-24T03%3A17%3A55.000Z). My result is normal (The scene has snow depth, which makes it appear noisy).

<img width="2547" height="1191" alt="Image" src="https://github.com/user-attachments/assets/e376c529-84b1-4f9a-ae42-e09cfe373a12" />
